templates/triton_services/README.ipynb

```diff
@@ -0,0 +1,334 @@
+    "!gcloud storage cp /tmp/workspace/diffusion-models/stable_diffusion_1_5/1/1.5-engine-batch-size-1/ $ANYSCALE_ARTIFACT_STORAGE/triton_model_repository/stable_diffusion_1_5/1/1.5-engine-batch-size-1/ --recursive"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run Triton Server locally on Anyscale Workspaces\n",
+    "\n",
+    "The `Deployment.py` is included in this workspace for you. This file demonstrates using\n",
+    "remote model repository to start triton serve, loading the specific Stable Diffusion\n",
+    "model, run inference with Triton, and serving the response through Ray Serve.\n",
+    "You can view the file by clicking on the `Deployment.py` in the file explorer. In\n",
+    "addition, you can also do prompt engineering, apply business logics, or doing model\n",
+    "composition with Ray Serve before returning the response as image. Run the follow code\n",
+    "to start Triton Server with Ray Serve."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!serve run deployment:triton_deployment --non-blocking"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A GPU worker node will be started by autoscaler. The model will be downloaded from the\n",
+    "cloud storage location where we just uploaded the model artifacts, and then loaded into\n",
+    "the Triton Server and serve the endpoint via Ray Serve. It might take few minutes to\n",
+    "start the server. \n",
+    "\n",
+    "Once you see the message \"Deployed app 'default' successfully.\". You can run the\n",
+    "following command to query the endpoint and save the image to a local file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!curl \"http://localhost:8000/generate?prompt=dogs%20in%20new%20york,%20realistic,%204k,%20photograph\" > dogs_photo.jpg"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An example of generated image will look like the following\n",
+    "\n",
+    "<img src=\"assets/dogs_photo.jpg\"/>\n",
+    "\n",
+    "## Deploy the application on Anyscale Services\n",
+    "\n",
+    "The `config.yaml` is also included in the workspace for you. You can view the file by\n",
+    "clicking on the `config.yaml` in the file explorer. Once you completed local\n",
+    "development on the workspace and ready to move to production, you can deploy the\n",
+    "service onto Anyscale Services by running the following command.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!anyscale service deploy -f config.yaml --name \"triton-stable-diffusion\""
```

Comment:
no need for the config file, can just do:
```suggestion
    "!anyscale service deploy --name \"triton-stable-diffusion\" deployment:triton_deployment"
```

this way it mirrors the `serve run` cmd too :)